### ISSUE TITLE
line numbering

### ISSUE THREAD

**Issue by foudfou** [sebsauvage/ZeroBin#54]
line numbering of the paste would be great.

**Comment by indigo423** [comment]
yes agree, one of the main features to allow discussion about code or configuration through via IRC.
